app/database_etl/summary_report_generator/summary_report.py

```diff
@@ -40,24 +50,58 @@ def get_human_readable_time(self, timestamp: float):
         dt = datetime.datetime.fromtimestamp(timestamp)
         return dt.strftime('%Y-%m-%d %H:%M:%S')
 
+    def get_table_counts_diff(self, before_counts, after_counts):
+        before_series = pd.Series(data=before_counts,
+                                  index=before_counts.keys(),
+                                  name='Table counts before')
+
+        after_series = pd.Series(data=after_counts,
+                                 index=after_counts.keys(),
+                                 name='Table counts after')
+        df = pd.concat([before_series, after_series], axis=1)
+
+        fig, ax = plt.subplots(figsize=(22, 11))
+        ax.set_axis_off()
+        table = ax.table(
+            cellText=df.values,
+            rowLabels=df.index,
+            colLabels=df.columns,
+            cellLoc='center',
+            loc='center right')
+
+        for (row, col), cell in table.get_celld().items():
+            if (row == 0) or (col == -1):
+                cell.set_text_props(fontproperties=FontProperties(weight='bold'))
+                cell.set_facecolor('powderblue')
+
+        table.set_fontsize(32)
+        table.scale(1, 4)
+        plt.tight_layout()
+        plt.savefig('etl_summary_report.png')
+        return
+
     def send_summary_report(self):
         body = f"Summary report for ETL run at {self.get_human_readable_time(self.start_time)}:\n\n"
-        body += f"Total runtime: {self.get_elapsed_time()} seconds\n"
+        body += f"Total runtime: {self.get_elapsed_time()} minutes\n"
         if hasattr(self, 'num_airtable_records'):
             body += f"Number of airtable records queried: {self.num_airtable_records}\n"
-        body += f"Table counts before: " + f"```{json.dumps(self.table_counts_before, indent=4)}```\n"
+        if hasattr(self, 'num_test_adjusted_records'):
+            body += f"Number of estimates test adjusted: {self.num_test_adjusted_records}\n"
+        if hasattr(self, 'num_divergent_estimates'):
+            body += f"Number of divergent test adjusted estimates: {self.num_divergent_estimates}\n"
         if hasattr(self, 'table_counts_after'):
-            body += f"Table counts after: " + f"```{json.dumps(self.table_counts_after, indent=4)}```\n"
             body += f"Status: SUCCESS"
+            self.get_table_counts_diff(self.table_counts_before, self.table_counts_after)
+            upload_slack_file(filename='etl_summary_report.png', caption=body)
+            os.remove('etl_summary_report.png')
         else:
-            body += f"Status: FAIL"
-        send_slack_message(body, channel="#dev-logging-etl")
+            body += f"Status: FAIL\n"
+            body += f"Table counts before: " + f"```{json.dumps(self.table_counts_before, indent=4)}```\n"
```

Comment:
worth it to put this into a table as well or nah?

Reply:
ya i can put it into a one column table just for consistency